src/server/database/Database/DatabaseWorkerPool.cpp

```diff
@@ -40,11 +40,11 @@
 #endif
 
 #if MARIADB_VERSION_ID >= 100600
-#define MIN_MYSQL_SERVER_VERSION 100500u
+#define MIN_MYSQL_SERVER_VERSION 100400u
 #define MIN_MYSQL_CLIENT_VERSION 30203u
 #else
-#define MIN_MYSQL_SERVER_VERSION 50700u
-#define MIN_MYSQL_CLIENT_VERSION 50700u
+#define MIN_MYSQL_SERVER_VERSION 50500u
+#define MIN_MYSQL_CLIENT_VERSION 50500u
```

Comment:
As explained in the issue, this will not be changed from 50700u.

AC cannot support MySQL 5.5 alongside MySQL 8.0